dev/new-rule/new-rule.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable node/no-unsupported-features/node-builtins */
+/* eslint-disable prefer-template */
+
+/**
+node.js-CLI-based generation of new rule template files
+Example usage: 
+--------------
+Templates use the rule name `my-new-rule`, so this example
+creates a functioning/immediately testable rule
+# /path/to/ember-template-lint
+$ yarn new my-new-rule
+$ yarn test my-new-rule
+*/
+
+const fs = require('fs');
+const path = require('path');
+
+// CLI Argument: `my-new-rule` (name of new rule to add)
+let newRuleName = process.argv[2];
+
+// Utility Filesystem: template files co-localized with this script
+let pathUtil = __dirname;
+let pathDocTemplate = path.join(pathUtil, 'new-rule-doc-template.md');
+let pathRuleTemplate = path.join(pathUtil, 'new-rule-template.js');
+let pathTestTemplate = path.join(pathUtil, 'new-rule-test-template.js');
+
+// Parent Filesystem: path/to/ember-template-lint
+let pathRoot = process.cwd();
```

Comment:
this should probably use `__dirname` too to be more robust :)

Reply:
sure! learning moment for me- how does it make it more robust?

Reply:
if you ran `yarn new foo` from a subfolder of the project it would create the files relative to that subfolder. but we do want it relative to the root folder of the project instead, which we can do via `path.join(__dirname, '..')` or something roughly like that.